packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ

```diff
@@ -0,0 +1,403 @@
+#import "@preview/touying-pres-ustc:0.2.0": *
+
+#let s = register(aspect-ratio: "16-9")
+#let s = (s.methods.numbering)(self: s, section: "1.", "1.1")
+#let s = (s.methods.info)(
+  self: s,
+  title: [Typst template for University of Science and Technology of China],
+  subtitle: [Continuously Improving...],
+  author: [Quaternijkon],
+  date: datetime.today(),
+  institution: [School of Computer Science and Technology, USTC],
+  logo: image("../../assets/img/USTC_logo_side.svg", width: 50%),
+  head-logo: image("../../assets/img/ustc_logo_side.svg",width: 20%),
```

Comment:
These two files can't be accessed from your template. You should either copy these images to `content/example/img` and reference this copy, or export `#let ustc-logo = image("/assets/img/USTC_logo_side.svg")` from the package and use that variable here.

Reply:
Yes, they can configure the `image` as they wish, replace the file by another one, or even wrap the image in some other element if they want. It is very flexible.